akvo/rsr/tests/rest/test_user.py

```diff
@@ -270,3 +255,46 @@ def test_user_with_programs(self):
         content = response.data
         program_ids = sorted([p['id'] for p in content['programs']])
         self.assertEqual(sorted([program_a.id, program_b.id]), program_ids)
+
+
+class OrganisationPasswordPolicyTestCase(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.org = self.create_organisation('Akvo')
+        policy = PolicyConfig.objects.create(name='Test policy', min_length=4, uppercases=2, symbols=2)
+        self.org.password_policy = policy
+        self.org.save()
+
+        email = 'foo@example.com'
+        self.initial_password = 'password'
+        self.user = self.create_user(email, self.initial_password)
+        self.make_employment(self.user, self.org, 'Users')
```

Comment:
What will happen with multiple employments? Are they possible? Is there a notion of "master" or "main" employment?

Reply:
Fixed, simulate multiple organisations, test the resolve_password_policy function.